### PR TITLE
[EASI-4240, EASI-4239] Operational Solutions Table Updates

### PR DESCRIPTION
**Pull request by garyjzhao**
# EASI-4240, EASI-4239

## Changes and Description

- Moved `solutions` column to far left
- Changed `must finish by` to `must be ready by`

## How to test this change

1. Navigate to operaitonal needs and solution page and verify changes

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: src/views/ModelPlan/TaskList/ITSolutions/Home/operationalNeedsTable.tsx [hunk 1]
```diff
@@ -437,11 +437,16 @@ const OperationalNeedsTable = ({
                     className="table-header"
                     scope="col"
                     style={{
-                      minWidth: '138px',
+                      minWidth: index !== 3 ? '138px' : '',
                       paddingBottom: '.5rem',
                       position: 'relative',
                       paddingLeft: index === 0 ? '.5em' : '0px',
-                      width: index === 5 ? '235px' : 'auto'
+                      // width: index === 5 ? '235px' : 'auto'
```
Comment (patrickseguraoddball) [inline src/views/ModelPlan/TaskList/ITSolutions/Home/operationalNeedsTable.tsx:444]: Can probably remove this commented out code

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
This table column became squished.  Should be a small fix

Screenshot 2024-04-24 at 9 22 21 AM

File: src/views/ModelPlan/TaskList/ITSolutions/Home/operationalNeedsTable.tsx [hunk 2]
```diff
@@ -437,11 +437,15 @@ const OperationalNeedsTable = ({
...
-                      minWidth: '138px',
+                      minWidth: index !== 3 ? '138px' : '',
                       paddingBottom: '.5rem',
                       position: 'relative',
                       paddingLeft: index === 0 ? '.5em' : '0px',
-                      width: index === 5 ? '235px' : 'auto'
+                      width:
+                        (index === 5 && '235px') ||
+                        (index === 2 && '170px') ||
+                        (index === 3 && type !== 'possibleNeeds' && '100px') ||
```
Comment (garyjzhao) [inline src/views/ModelPlan/TaskList/ITSolutions/Home/operationalNeedsTable.tsx:447]: image

**Review by patrickseguraoddball — APPROVED** [review]
LG

**Pull request merged**